<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationsFrameFrame</class>
 <widget class="QFrame" name="AnimationsFrameFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QFrame" name="animationsFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="animationsTitleLayout">
         <item>
          <widget class="QLabel" name="animationsTitle">
           <property name="text">
            <string>Animations</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="animationsTitleSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="animationsAddButton">
           <property name="toolTip">
            <string>Add new animation</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../iconset.qrc">
             <normaloff>:/src/icons/16x16/actions/list-add.png</normaloff>:/src/icons/16x16/actions/list-add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="animationsRemoveButton">
           <property name="toolTip">
            <string>Remove current animation</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../iconset.qrc">
             <normaloff>:/src/icons/16x16/actions/list-remove.png</normaloff>:/src/icons/16x16/actions/list-remove.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="AnimationListView" name="animationsList"/>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="animationsFrameFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="animationsFrameTitleLayout">
         <item>
          <widget class="QLabel" name="animationsFrametitle">
           <property name="text">
            <string>Animations Frames</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="AnimationsFrameListView" name="animationsFramesList"/>
       </item>
      </layout>
      <zorder>animationsFramesList</zorder>
      <zorder>animationsFramesList</zorder>
      <zorder>horizontalLayoutWidget_2</zorder>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimationListView</class>
   <extends>QListView</extends>
   <header>src/animationlistview.h</header>
  </customwidget>
  <customwidget>
   <class>AnimationsFrameListView</class>
   <extends>QListView</extends>
   <header>src/animationsframelistview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../iconset.qrc"/>
 </resources>
 <connections/>
</ui>
